pom.xml

```diff
@@ -203,17 +195,6 @@
         <version>${protobuf.version}</version>
         <scope>test</scope>
       </dependency>
-      <dependency>
-        <groupId>junit</groupId>
-        <artifactId>junit</artifactId>
-        <version>${junit.version}</version>
-        <scope>test</scope>
-      </dependency>
-      <dependency>
-        <groupId>org.apache.tomcat.embed</groupId>
-        <artifactId>tomcat-embed-core</artifactId>
-        <version>${tomcat-embed-core.version}</version>
-      </dependency>
```

Comment:
hmmm... shall we really remove that overriding? I have concerns, that the next security finding will occur in the same libraries... especially tomcat has very frequent new issues...

Reply:
Good point. I'll leave it as it was.